<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtVolumeConfiguration</class>
 <widget class="QDialog" name="LXQtVolumeConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Control Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device to control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="alsaRadioButton">
          <property name="text">
           <string>ALSA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pulseAudioRadioButton">
          <property name="text">
           <string>PulseAudio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ossRadioButton">
          <property name="text">
           <string>OSS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="devAddedCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="muteOnMiddleClickCheckBox">
        <property name="text">
         <string>Mute on middle click</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreMaxVolumeCheckBox">
        <property name="text">
         <string>Allow volume beyond 100% (0dB)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alwaysShowNotificationsCheckBox">
        <property name="text">
         <string>Always notify about volume changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showKeyboardNotificationsCheckBox">
        <property name="text">
         <string>Notify about volume changes with keyboard</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Volume adjust step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="stepSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>External Mixer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="mixerLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>LXQtVolumeConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>LXQtVolumeConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
